.travis.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       script:
         - yarn pre-test
         - yarn cov-test --forbid-only
+        - ./scripts/install-chromedriver.sh
```

Comment:
MINOR: installing chromedriver should be the first step b/c we might use it in pre-test or cov-test in the future

Reply:
@ninok Finally i've skipped this script and chosen to use `DETECT_CHROMEDRIVER_VERSION` introduced recently.
Tihs needs update too, so it's hard to tell if the feature works though - see new version of patch,